src/lib/sort-rules.js

```diff
@@ -1,9 +1,27 @@
 'use strict'
 
 function getSortedRules(rules) {
-  return rules.sort(function sort(a, b) {
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(rules) {
+  var sortedRules = []
+
+  Object.keys(rules)
+    .sort(sortAlphabetically)
+    .forEach(function map(ruleName) {
+      sortedRules.push(ruleName, rules[ruleName].config1, rules[ruleName].config2)
```

Comment:
Making the module's output to read

```JavaScript
[{ 'foo-rule': {config1: '0 foo', config2:  '2 foo'} }, ....]
```

sounds good, let that `cliui util` do the necessary flattening.